app/views/header/_header.slim

```diff
@@ -0,0 +1,12 @@
+.header-container 
+    .logo-search-bar
+        a href = "/"
```

Comment:
prefer using link to
```
link_to root_path do
```

Reply:
![image](https://user-images.githubusercontent.com/62087110/206620153-4cbccbae-71d9-454b-b643-7958961b433a.png)